tools/scan-build-py/libscanbuild/analyze.py

```diff
@@ -639,7 +639,8 @@ def dispatch_ctu(opts, continuation=run_analyzer):
                 + opts['flags'] + [opts['file']]
             triarch = get_triple_arch(cmd, cwd)
             ctu_options = ['ctu-dir=' + os.path.join(ctu_config.dir, triarch),
-                           'reanalyze-ctu-visited=true']
+                           'reanalyze-ctu-visited=true',
+                           'experimental-enable-naive-ctu-analysis=true']
```

Comment:
Are we sure that setting it to constant on is what we want? Was it tested accordingly?

Reply:
this is set only when ctu_config.analyze is set. Test files were updated but they call clang directly so this parameter was added to the line containing the lit running command.

Reply:
Ok, so this means that this experimental mode will be the default for everybody running CTU with scan-build-py